### ISSUE TITLE
Drop 'os' and 'ver' from the AWS_AMIS config

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/fedimg#38]
The config has a list of regions to upload to, which is called AWS_AMIs internally.  It is a pipe-delimited list of regions with corresponding AMIs.  For unknown reasons, each row contains an 'os' entry and a 'ver' entry (for example 'RHEL' and '6.5').  From what I can tell, they are not used.  Can they just be removed?

**Comment by Amitgb14** [comment]
Drop from ```/etc/fedimg.cfg``` or (https://github.com/fedora-infra/fedimg/blob/develop/fedimg/services/ec2.py#L99) and (https://github.com/fedora-infra/fedimg/blob/develop/fedimg/services/ec2.py#L100) .

**Comment by ralphbean** [comment]
To finish cleaning this up, we'll need to remove the extra unnecessary info from [this file](https://infrastructure.fedoraproject.org/cgit/ansible.git/tree/roles/fedimg/templates/fedimg.cfg#n35) once the patch from #46 has been released and deployed.

**Comment by sudheesh001** [comment]
@ralphbean So I was just looking to get started on `fedimg` and according to this, do I have to remove this from the given file ?
```
amis = us-east-1|RHEL|6.5|x86_64|ami-be6a98d6|aki-919dcaf8
       ap-northeast-1|RHEL|6.5|x86_64|ami-e7aee0e6|aki-176bf516
       ap-southeast-1|RHEL|6.5|x86_64|ami-c683df94|aki-503e7402
       ap-southeast-2|RHEL|6.5|x86_64|ami-41ra8f7b|aki-c362fff9
       eu-central-1|RHEL|6.5|x86_64|ami-76eddb6b|aki-184c7a05
       eu-west-1|RHEL|6.5|x86_64|ami-81f23cf6|aki-52a34525
       sa-east-1|RHEL|6.5|x86_64|ami-b7ec43aa|aki-5553f448
       us-west-1|RHEL|6.5|x86_64|ami-fc393eb9|aki-880531cd
       us-west-2|RHEL|6.5|x86_64|ami-79daa849|aki-fc8f11cc
       us-east-1|RHEL|6.5|i386|ami-acac51c4|aki-8f9dcae6
       ap-northeast-1|RHEL|6.5|i386|ami-c7bff1c6|aki-136bf512
       ap-southeast-1|RHEL|6.5|i386|ami-9eb8e4cc|aki-ae3973fc
       ap-southeast-2|RHEL|6.5|i386|ami-87f194bd|aki-cd62fff7
       eu-central-1|RHEL|6.5|i386|ami-76eddb6b|aki-184c7a05
       eu-west-1|RHEL|6.5|i386|ami-7101cf06|aki-68a3451f
       sa-east-1|RHEL|6.5|i386|ami-e9e847f4|aki-5b53f446
       us-west-1|RHEL|6.5|i386|ami-eacfc8af|aki-8e0531cb
       us-west-2|RHEL|6.5|i386|ami-25cab815|aki-f08f11c0
```

**Comment by sayanchowdhury** [comment]
This has been fixed in #62
